lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -572,7 +573,7 @@ spec = do
             , expectErrorMessage $ errMsg404NoWallet (w ^. walletId)
             ]
 
-    describe "STAKE_POOLS_ESTIMATE_FEE_04 - wallet ids" $ do
+    describe "STAKE_POOLS_ESTIMATE_FEE_04 - invalid wallet ids" $ do
```

Comment:
So the wallet ids are either for a non-existing wallet, or totally wrong.

Maybe "false wallet ids" would be better (consistent with the `falseWalletIds` name) 🤷‍♂

Reply:
Went with "false wallet ids".